docs/Policies/wulver_policies.md

```diff
@@ -1,39 +1,39 @@
 
 ## Shared Condo Partnership
 
 Faculty who routinely need more resources than the initial allocation may buy nodes and contribute to the cluster. A catalog of select hardware for inclusion in the cluster will be made available. The PI and associated users will be able to submit jobs with a higher priority up to the resources contributed. Note that these jobs may or may not run on the actual nodes purchased. The allocated resources will be available via a floating reservation for the amount of resources purchased. Contributors will be able to additionally submit jobs using SUs as well as lower priority. The university will subsidize all infrastructure costs for these nodes. This floating reservation will be available for five years.
 
 ## Private Pool
 
-If the shared condo module does not satisfy the needs of the PI, a private pool may be set up. In addition to the nodes, the PI will be charged for all infrastructure costs, including but not limited to electricity, HVAC, system administration, etc…. It is strongly recommended to first try the shared condo model. If the shared condo model does not work, the nodes can be converted to a private pool.
+If the shared condo module does not satisfy the needs of the PI, a private pool may be set up. In addition to the nodes, the PI will be charged for all infrastructure costs, including but not limited to electricity, HVAC, system administration, etc. It is strongly recommended to first try the shared condo model. If the shared condo model does not work, the nodes can be converted to a private pool.
 
 ## Job Priorities
 
 * Standard Priority
-    * Charged against the SU allocation.
-    * Wall time maximum - 72 hours.
-    * Jobs are not preemptable.
+    * Faculty PIs are allocated 300,000 Service Units (SU) per year on request at no cost
+    * Wall time maximum - 72 hours
+    * Additional SUs may be purchased at a cost of $0.01/SU
+    * The minimum purchase is 50,000 SU ($500)
+
 * Low Priority
     * Not charged against SU allocation
     * Wall time maximum - 72 hours
 
-Jobs are preemptable by higher priority jobs
-
 * High Priority
     * Not charged against SU allocation
-    * Wall time maximum is determined by PI; 3 days are suggested.  ARCS reserves right to reboot nodes once a month for maintenance (perhaps first Monday of the month)
-    * Jobs are not preemptable
+    * Wall time maximum - 72 hours
```

Comment:
Please revert to previous version
Wall time: 72 hours (default), PI can request longer walltimes up to 30 days. ARCS reserves right to reboot nodes once a month for maintenance - second Tuesday of each month.